### PR TITLE
tools/xz: update to 5.2.10

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Update to latest version.

### CODE REVIEW

File: tools/xz/Makefile [hunk 1]
```diff
@@ -7,12 +7,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xz
-PKG_VERSION:=5.2.9
+PKG_VERSION:=5.2.10
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
```
Comment (lovehackintosh) [inline tools/xz/Makefile:12]: Hi,I think switch to the xz tarball will save a little bit of traffic.What do you think of it?

Comment (PolynomialDivision) [reply tools/xz/Makefile]: Yeah we can do it. Switched to it. Thanks.

Comment (hauke) [reply tools/xz/Makefile]: don't we need xz to unpack xz archives? I think xz is not pretty common, but I would prefer to take the bz2 or gz version

Comment (lovehackintosh) [reply tools/xz/Makefile]: Hi,now, most Linux distributions install xz by default, such as Ubuntu >= 14.06, Centos >= 6, Arch Linux and Debian，and can unpack xz archives.What do you think?

Comment (PolynomialDivision) [reply tools/xz/Makefile]: > don't we need xz to unpack xz archives? I think xz is not pretty common, but I would prefer to take the bz2 or gz version

I don't care. ;) I just need some decision.

Comment (hnyman) [reply tools/xz/Makefile]: This was earlier explicitly changed back from .xz to .bz2 by the commit e68c0a1325f8 just for the reason that we can't assume xz support on all build hosts.

Comment (PolynomialDivision) [reply tools/xz/Makefile:12]: Changed that back.

**Pull request merged**